experimental/packages/opentelemetry-instrumentation/src/instrumentation.ts

```diff
@@ -79,6 +80,15 @@ implements types.Instrumentation {
       this.instrumentationName,
       this.instrumentationVersion
     );
+
+    this._setMeterInstruments();
+  }
+
+  /**
+   * Sets the new meter instruments with the current Meter Provider.
+   */
+  protected _setMeterInstruments(): void {
```

Comment:
The term "metric instrument" is more commonly used. Also, I'd find that "set" is used when the value is specified by the caller. Here, the instruments are rather updated by the instrumentation itself. So I'd suggest a naming like:

```suggestion
  protected _updateMetricInstruments(): void {
```